### PR TITLE
Update serverless.md

### PR DESCRIPTION
**Pull request by chspy**
NOTE: Please read the Contributing.md guidelines before submitting your patch:

https://github.com/apache/incubator-apisix/blob/master/Contributing.md#how-to-add-a-new-feature-or-change-an-existing-one

### Summary

SUMMARY_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #XXX
命令行示例语句错误：data多了一层plugins

### CODE REVIEW

**Comment by moonming** [comment]
please also update the Chinese version, thx

**Comment by moonming** [comment]
@chspy you should escape single quotes in curl, please refer to https://github.com/apache/incubator-apisix/issues/969

**Comment by moonming** [comment]
@chspy any update?

File: doc/plugins/serverless.md [hunk 1]
```diff
@@ -73,12 +73,10 @@ curl -i http://127.0.0.1:9080/apisix/admin/routes/1 -X PUT -d '
...
     "plugins": {
-        "plugins": {
-            "serverless-pre-function": {
-                "phase": "rewrite",
-                "functions" : ["return function() ngx.log(ngx.ERR, 'serverless pre function'); end"]
-            }
-        },
+        "serverless-pre-function": {
+            "phase": "rewrite",
+            "functions" : ["return function() ngx.log(ngx.ERR, 'serverless pre function'); end"]
```
Comment (membphis) [inline doc/plugins/serverless.md]: that is wrong.

correct way: `"return function() ngx.log(ngx.ERR, \"serverless pre function\"); end"`

**Comment by moonming** [comment]
merged first, and I will create another PR to fix.

**Pull request merged**